weblogic-azure-aks/src/main/arm/scripts/setupDBConnections.sh

```diff
@@ -0,0 +1,322 @@
+}
+
+function wait_for_operation_completed() {
+    # Make sure all of the pods are running.
+    replicas=$(kubectl -n ${wlsDomainNS} get domain ${wlsDomainUID} -o json \
+        | jq '. | .spec.clusters[] | .replicas')
+
+    utility_wait_for_pod_restarted \
+        ${timestampBeforePatchingDomain} \
+        ${replicas} \
+        ${wlsDomainUID} \
+        ${checkPodStatusMaxAttemps} \
+        ${checkPodStatusInterval}
+
+    utility_wait_for_pod_completed \
+        ${replicas} \
+        ${wlsDomainNS} \
+        ${checkPodStatusMaxAttemps} \
+        ${checkPodStatusInterval}
+}
+
+function delete_datasource() {
+    echo "remove secret and model of data source ${jdbcDataSourceName}"
+    # remove secret
+    # remove model
+    chmod ugo+x $scriptDir/dbUtility.sh
+     echo "${dbPassword}" | \
+        bash $scriptDir/dbUtility.sh \
+        ${databaseType} \
+        "${dbUser}" \
+        "${dsConnectionURL}" \
+        "${jdbcDataSourceName}" \
+        "${wlsDomainUID}" \
+        "${dbSecretName}" \
+        "${optTypeDelete}"
+
+    # update weblogic domain
+    remove_datasource_from_domain
+
+    wait_for_operation_completed
+}
+
+function validate_datasource() {
+    dsScriptFileName=get-datasource-status.py
+    testDatasourceScript=${scriptDir}/${dsScriptFileName}
+    podNum=$(kubectl -n ${wlsDomainNS} get pod -l weblogic.clusterName=${wlsClusterName} -o json | jq '.items| length')
+    if [ ${podNum} -le 0 ]; then
+        echo "Ensure your cluster has at least one pod."
+        exit 1
+    fi
+
+    podName=$(kubectl -n ${wlsDomainNS} get pod -l weblogic.clusterName=${wlsClusterName} -o json \
+        | jq '.items[0] | .metadata.name' \
+        | tr -d "\"")
+
+    # get non-ssl port
+    clusterTargetPort=$(kubectl get svc ${wlsClusterSvcName} -n ${wlsDomainNS} -o json | jq '.spec.ports[] | select(.name=="default") | .port')
+    t3ConnectionString="t3://${wlsClusterSvcName}.${wlsDomainNS}.svc.cluster.local:${clusterTargetPort}"
+    cat <<EOF >${testDatasourceScript}
+connect('${wlsUser}', '${wlsPassword}', '${t3ConnectionString}')
+serverRuntime()
+print 'start to query data source jndi bean'
+dsMBeans = cmo.getJDBCServiceRuntime().getJDBCDataSourceRuntimeMBeans()
+ds_name = '${jdbcDataSourceName}'
+for ds in dsMBeans:
+    if (ds_name == ds.getName()):
+        print 'DS name is: '+ds.getName()
+        print 'State is ' +ds.getState()
+EOF
+
+    echo "copy test script ${testDatasourceScript} to pod path /tmp/${dsScriptFileName}"
+    targetDSFilePath=/tmp/${dsScriptFileName}
+    kubectl cp ${testDatasourceScript} -n ${wlsDomainNS} ${podName}:${targetDSFilePath}
+    kubectl exec -it ${podName} -n ${wlsDomainNS} -- bash -c "wlst.sh ${targetDSFilePath}" | grep "State is Running"
```

Comment:
You should probably name the container "-c weblogic-server" just in case there are other containers in the pod for the server instance (e.g. fluentd or Istio proxy, etc.)

Reply:
Fixed it in https://github.com/galiacheng/weblogic-azure/commit/17537a088bc845f9cfe9d21f8d55949a5bcab4ea